Automattic/wp-calypso
# Update Stylelint Scales package

#### Changes proposed in this Pull Request

* Updates to the latest version of Stylelint Scales for error message clarity

#### Testing instructions

* Switch to this PR
* `yarn start`
* Try to commit a change to a .scss file that includes a `font-size` value outside our type scale, ie. `font-size: 36px` or `font-size: 1.832rem`
* You should get an error like this one, specifying the required scale and unit(s):

<img width="571" alt="Screen Shot 2020-08-18 at 2 26 42 PM" src="https://user-images.githubusercontent.com/2124984/90551289-18a79500-e15f-11ea-99d8-68935dec2d52.png">

* Try switching to a value on the scale in `rem`s; you should be able to commit successfully.
